extension/sigv4authextension/README.md

```diff
@@ -0,0 +1,51 @@
+# Authenticator - Sigv4
+
+This extension provides the Sigv4 process of adding authentication information to AWS API requests to HTTP based exporters for AWS services. This extension implements the `configauth.ClientAuthenticator` interface. For more information on the Sigv4 process, please look [here](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html).
+
+The authenticator type has to be set to `sigv4auth`. 
+
+## Configuration
+
+The configuration fields are as follows:
+
+* `region`: **Required**. The AWS region for AWS Sigv4
+    * Note that an attempt will be made to obtain a valid default from the `AWS_DEFAULT_REGION` environment variable
+    * [List of AWS regions](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html)
+* `service`: **Required**. The AWS service for AWS Sigv4
+* `role_arn`: **Optional**. The Amazon Resource Name (ARN) of a role to assume
+* `sts_endpoint`: **Optional**. The endpoint for the `role_arn`
```

Comment:
The STS endpoint for querying credentials for `role_arn`. Document the accepted values here and the default (I guess default is to use AWS SDK Go's defaults)